### PR TITLE
Issue #445 Document supplying pull secret on `crc start`

### PR DESCRIPTION
**Pull request by robin-owen**
We may also want to document the usage of the `--pull-secret-file` flag. This has not been added to the documentation yet. Please let me know what you think and I'll update this PR accordingly.

### CODE REVIEW

**Comment by anjannath** [comment]
@robin-owen maybe using the `--pull-secret-file | -p` flag in `crc start` should be the suggested way, and as an alternative, if the user doesn't supply it he'll be prompted for it during start.

File: docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc [hunk 1]
```diff
@@ -8,6 +8,13 @@ The [command]`{bin} start` command starts the {prod} virtual machine and OpenShi
 * The host machine has been set up using the [command]`{bin} setup` command.
 For more information, see <<setting-up-codeready-containers_{context}>>.
 * A valid OpenShift system bundle with the `.crcbundle` extension.
+* A valid OpenShift user pull secret.
+The pull secret can be copied or downloaded from the Pull Secret section of the link:https://cloud.redhat.com/openshift/install/metal/user-provisioned[Install on Bare Metal: User-Provisioned Infrastructure] page on cloud.redhat.com.
```
Comment (gbraad) [inline docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc]: Take note: this is at the moment right, but when our landing page is publshed, tis has to change.

**Comment by gbraad** [comment]
@anjannath that can be handled in a follow-up issue.
This information is correct and is necessary.

I will merge, but hope to get an additional PR about the `crc config set pull-secret` too.

**Comment by gbraad** [comment]
got accidently closed.

File: docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc (hunk 1 above)
Comment (robin-owen) [reply docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc]: Already noted. :)

File: docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc [hunk 2]
```diff
@@ -8,6 +8,13 @@ The [command]`{bin} start` command starts the {prod} virtual machine and OpenShi
...
++
+[NOTE]
+====
+A Red Hat account is required in order to access the user pull secret.
```
Comment (praveenkumar) [inline docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc]: 👍

**Comment by gbraad** [comment]
retest this please

**Pull request merged**